<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InitialPickupSettings</class>
 <widget class="QDialog" name="InitialPickupSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure starting pickups</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_Beams">
     <property name="title">
      <string>Beams</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox_IceBeam">
        <property name="text">
         <string extracomment="The Ice Beam">Ice</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_PowerBeam">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string comment="power-beam" extracomment="The Power Beam">Power</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <property name="tristate">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_WaveBeam">
        <property name="text">
         <string extracomment="The Wave Beam">Wave</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QCheckBox" name="checkBox_PlasmaBeam">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string extracomment="The Plasma Beam">Plasma</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="checkBox_SuperMissile">
        <property name="text">
         <string>Supermissile</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_ChargeBeam">
        <property name="text">
         <string extracomment="The Charge Beam">Charge</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="checkBox_WaveBuster">
        <property name="text">
         <string>Wavebuster</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="checkBox_IceSpreader">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Ice Spreader</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="checkBox_Flamethrower">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Flamethrower</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QCheckBox" name="checkBox_PhazonBeam">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string extracomment="The Phazon Beam">Phazon</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_Visors">
     <property name="title">
      <string>Visors</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_VisorCombat">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Combat</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_VisorScan">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string extracomment="The Scan Visor">Scan</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_VisorThermal">
        <property name="text">
         <string extracomment="The Thermal Visor">Thermal</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox_VisorXRay">
        <property name="text">
         <string extracomment="The X-Ray Visor">X-Ray</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string extracomment="Here we refer to &quot;Morph Ball upgrades&quot; as a whole">Morph Ball</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_BallMorph">
        <property name="text">
         <string extracomment="Here we refer to &quot;Morph Ball&quot; standalone upgrade">Morph Ball</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_BallSpider">
        <property name="text">
         <string>Spider Ball</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_BallBoost">
        <property name="text">
         <string>Boost Ball</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox_BallBombs">
        <property name="text">
         <string>Bombs</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="title">
      <string>Suits</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="3">
       <widget class="QCheckBox" name="checkBox_SuitPhazon">
        <property name="text">
         <string extracomment="The Phazon Suit">Phazon</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBox_SuitGravity">
        <property name="text">
         <string extracomment="The Gravity Suit">Gravity</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_SuitVaria">
        <property name="text">
         <string extracomment="The Varia Suit">Varia</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_SuitPower">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string comment="power-suit">Power</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string extracomment="Here we refer as &quot;movement upgrades&quot; or &quot;movement enhancement devices&quot;">Movement</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_SpaceJump">
        <property name="text">
         <string>Space Jump Boots</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="checkBox_GrappleBeam">
        <property name="text">
         <string>Grapple Beam</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="title">
      <string extracomment="Expansions or tanks that increment the limit of something">Expansions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string extracomment="Missile expansions">Missiles</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBox_Missiles">
        <property name="maximum">
         <number>250</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Energy tanks</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBox_ETanks">
        <property name="maximum">
         <number>14</number>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Power bombs</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QSpinBox" name="spinBox_PowerBombs">
        <property name="maximum">
         <number>8</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>InitialPickupSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>InitialPickupSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>306</x>
     <y>484</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_SuitPhazon</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBox_PhazonBeam</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>431</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>67</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
